model_lab_configs/huggingface/Intel/bert-base-uncased-mrpc/1/bert_ov.json.config

```diff
@@ -11,14 +12,58 @@
         "name": "Evaluate on",
         "type": "enum",
         "displayNames": [
+            "Intel CPU",
+            "Intel GPU",
             "Intel NPU"
         ],
-        "path": "systems.local_system.accelerators.0.execution_providers.0",
+        "path": "systems.local_system.accelerators.0.device",
         "values": [
-            "OpenVINOExecutionProvider"
+            "cpu",
+            "gpu",
+            "npu"
         ],
         "readOnly": false
     },
+    "runtimeInConversion": {
+        "autoGenerated": true,
+        "name": "Target Device in Conversion",
```

Comment:
I think "Target Device" is enough. This will be shown in General, and applies both for conversion and quantization